### PR TITLE
OCD-3824: Remove transparency attestation

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: changes/ocd-3824.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+DROP VIEW IF EXISTS openchpl.developers_with_attestations;
+DROP VIEW IF EXISTS openchpl.certified_product_details;
+
+-- delete tables
+DROP TABLE IF EXISTS openchpl.acb_vendor_map;
+ALTER TABLE openchpl.pending_certified_product DROP COLUMN IF EXISTS vendor_transparency_attestation;
+
+-- delete columns
+-- dropping the type by both names (can remove first line below after OCD-3717 goes to STG)
+DROP TYPE IF EXISTS openchpl.attestation;
```
Comment (andlar) [inline changes/ocd-3824.sql]: Should this line be removed?

Comment (kekey1) [reply changes/ocd-3824.sql]: I did put that comment in there. Maybe it should say when 3717 goes to PROD? At this point I don't think I would know which order the ocd sql files would be executed.

Comment (andlar) [reply changes/ocd-3824.sql]: Maybe we leave this here as is, and evaluate based on when 3717 goes to PROD and whether this one goes with it or not

Comment (kekey1) [reply changes/ocd-3824.sql]: Works for me

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by tmy1313 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**